NAMESPACE

```diff
@@ -1,5 +1,3 @@
-# Generated by roxygen2: do not edit by hand
```

Comment:
You need to update your R packages and rebuild this package: [`update.packages(ask = FALSE, checkBuilt = TRUE)`](https://yihui.name/en/2017/05/when-in-doubt-upgrade/), because I have updated Rd2roxygen so that this comment is no longer removed.